docs/installation/requirements.md

```diff
@@ -28,8 +39,10 @@ For more information on which OSs were tested with Rancher managed K3s clusters,
 
 Hardware requirements scale based on the size of your deployments. Minimum recommendations are outlined here.
 
-*    RAM: 512MB Minimum (we recommend at least 1GB)
-*    CPU: 1 Minimum
+| Spec | Minimum | Recommended |
+|------|---------|-------------|
+| CPU  | 1       | 2           |
```

Comment:
Sometimes people get confused by CPUs and ask if we mean core or socket; might be good to clarify
```suggestion
| CPU  | 1 core      | 2 cores          |
```